shared/js/otSdkDriver.js

```diff
@@ -776,6 +789,21 @@ loop.OTSdkDriver = (function() {
     },
 
     /**
+     * Sends the cursor position on the data channel.
+     *
+     * @param {String} message The message to send.
+     */
+    sendCursorMessage: function(message) {
+      if (!this._publisherChannel || !this._subscriberChannel) {
+        return;
```

Comment:
We must ensure that both users are ready to receive and send information. If an user has entered in the room but hasn't subscribed to the video stream yet, we should prevent sending it the cursor information because it will display while the stream video is still loading.